frontend/src/App.vue

```diff
@@ -43,10 +43,10 @@
 	--color-data-general-rgb: 197, 197, 197;
 	--color-data-general-dim: #767676;
 	--color-data-general-dim-rgb: 118, 118, 118;
-	--color-data-vector: #65bbe5;
-	--color-data-vector-rgb: 101, 187, 229;
-	--color-data-vector-dim: #4b778c;
-	--color-data-vector-dim-rgb: 75, 119, 140;
```

Comment:
Is there another way that can avoid this change? It breaks the naming convention for the high-level data types people will know by name (raster, vector, bitmap, text, etc.) and also breaks the coloring used in the tool shelf (referenced in the `.svg` files).

Reply:
Raster and bitmap are the same thing aren't they? You can name the things whatever you like.

Reply:
Vector is also a confusing name as it represents both a mathematical vector (magnitude and direction) as well as a method of defining the geometry of an image.

Reply:
I'll rename this back to vector so we can merge this PR.